db/seeds.rb

```diff
@@ -153,97 +153,97 @@
+ extract_potential: nil, alpha_acid: 6.5, attenuation: nil, main_class: "Hop" )
 Ingredient.create(name: 'Northern Brewer', sub_type: 'England',
- extract_potential: nil, alpha_acid: 9, attenuation: nil, main_class: "Hop")
+ extract_potential: nil, alpha_acid: 9, attenuation: nil, main_class: "Hop" )
 Ingredient.create(name: 'Nugget', sub_type: 'U.S.',
- extract_potential: nil, alpha_acid: 13, attenuation: nil, main_class: "Hop")
+ extract_potential: nil, alpha_acid: 13, attenuation: nil, main_class: "Hop" )
 Ingredient.create(name: 'Palisade', sub_type: 'U.S.',
- extract_potential: nil, alpha_acid: 8, attenuation: nil, main_class: "Hop")
+ extract_potential: nil, alpha_acid: 8, attenuation: nil, main_class: "Hop" )
 Ingredient.create(name: 'Perle', sub_type: 'Germany',
- extract_potential: nil, alpha_acid: 8, attenuation: nil, main_class: "Hop")
+ extract_potential: nil, alpha_acid: 8, attenuation: nil, main_class: "Hop" )
 Ingredient.create(name: 'Pride of Ringwood', sub_type: 'Australia',
- extract_potential: nil, alpha_acid: 8.75, attenuation: nil, main_class: "Hop")
+ extract_potential: nil, alpha_acid: 8.75, attenuation: nil, main_class: "Hop" )
 Ingredient.create(name: 'Saaz', sub_type: 'Czech Republic',
- extract_potential: nil, alpha_acid: 3.75, attenuation: nil, main_class: "Hop")
+ extract_potential: nil, alpha_acid: 3.75, attenuation: nil, main_class: "Hop" )
 Ingredient.create(name: 'Rubin', sub_type: 'Czech Republic',
- extract_potential: nil, alpha_acid: 10.5, attenuation: nil, main_class: "Hop")
+ extract_potential: nil, alpha_acid: 10.5, attenuation: nil, main_class: "Hop" )
 Ingredient.create(name: 'Bohemie', sub_type: 'Czech Republic',
- extract_potential: nil, alpha_acid: 6.75, attenuation: nil, main_class: "Hop")
+ extract_potential: nil, alpha_acid: 6.75, attenuation: nil, main_class: "Hop" )
 Ingredient.create(name: 'Saphir', sub_type: 'Germany',
- extract_potential: nil, alpha_acid: 3.25, attenuation: nil, main_class: "Hop")
+ extract_potential: nil, alpha_acid: 3.25, attenuation: nil, main_class: "Hop" )
 Ingredient.create(name: 'Select', sub_type: 'Germany',
- extract_potential: nil, alpha_acid: 5, attenuation: nil, main_class: "Hop")
+ extract_potential: nil, alpha_acid: 5, attenuation: nil, main_class: "Hop" )
 Ingredient.create(name: 'Simcoe', sub_type: 'U.S.',
- extract_potential: nil, alpha_acid: 13, attenuation: nil, main_class: "Hop")
+ extract_potential: nil, alpha_acid: 13, attenuation: nil, main_class: "Hop" )
 Ingredient.create(name: 'Sorachi Ace', sub_type: 'Japan',
- extract_potential: nil, alpha_acid: 13, attenuation: nil, main_class: "Hop")
+ extract_potential: nil, alpha_acid: 13, attenuation: nil, main_class: "Hop" )
 Ingredient.create(name: 'Southern Cross', sub_type: 'New Zealand',
- extract_potential: nil, alpha_acid: 12.75, attenuation: nil, main_class: "Hop")
+ extract_potential: nil, alpha_acid: 12.75, attenuation: nil, main_class: "Hop" )
```

Comment:
Line is too long. [81/80]<br>Align the elements of a hash literal if they span more than one line.<br>Space inside parentheses detected.